.github/workflows/maven.yml

```diff
@@ -61,7 +61,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        java: [8, 11, 17]
+        java: [11, 17, 21]
```

Comment:
There is no way to build on 8 **and** 21?

Reply:
I must admit I haven't checked all the intermediate mockito versions, but the one currently in use does not work with JDK 21 and the most recent one is built with JDK 11 byte code.

Reply:
I can imagine the Mockito team 'made the jump' and dropped Java 8 altogether. This change would not _prevent_ running Maven on Java 8, would it?

Reply:
Definitely not, it's only the build time requirements that are raised, but the target byte code is still JDK 8.

Reply:
Thanks for clarifying!